messages/commands/debitpolicy.go

```diff
@@ -0,0 +1,21 @@
+package commands
+
+import "time"
+
+// CheckWithdrawalAllowedByDebitPolicy is a command requesting approval from the
+// debit policy for funds to be withdrawn from a bank account for a withdrawal.
+type CheckWithdrawalAllowedByDebitPolicy struct {
+	Timestamp     time.Time
```

Comment:
What is this the timestamp of? Might be if we name such timestamps very explicitly, after all they're part of our business domain.

Reply:
It is supposed to be the timestamp of the debit transaction. Does that exist at this location, or only after the policy approves the action? Also, I didn't know where to get the time value from yet.

I will change the name to `TransactionTimestamp`? Is `time.Time` the correct type to use?

Reply:
Which timestamp of the debit transaction? A transaction is a process that occurs over time, not an instant in time. We need to ask: what's the *business* rule about daily limits, which day do they apply to.

`time.Time` is the correct type, yep.

Reply:
Ok, I guess i need to think more about that. I assume the initial inception time of the transaction would be the correct time to use?

Reply:
Yeah, that makes sense to me.

Reply:
To be clear, I'm referring to the time at which the customer requested the transaction. 

I think, at least for the sake of this example, this would be the `time.Now()` inside the gRPC call that executes the `StartTransfer` (etc) command.

Reply:
So to do this as suggested, I will need to update the `Withdrawal` and `Transfer` commands to include the timestamp.